src/main/java/seedu/address/logic/commands/DepleteIngredientCommand.java

```diff
@@ -0,0 +1,115 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INDEX;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_INGREDIENT_QUANTITY;
+import static seedu.address.model.IngredientModel.PREDICATE_SHOW_ALL_INGREDIENTS;
+
+import java.util.List;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.commons.core.index.Index;
+import seedu.address.logic.CommandHistory;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.ingredient.Ingredient;
+import seedu.address.model.ingredient.IngredientQuantity;
+import seedu.address.model.person.exceptions.DuplicateItemException;
+
+/**
+ * Depletes the quantity of existing ingredient in the restaurant book.
+ */
+
+public class DepleteIngredientCommand extends Command {
```

Comment:
deplete and delete look very similar- consider having a different name for the command to be less confusing for the reader?